src/transport_ofi.c

```diff
@@ -1071,6 +1074,10 @@ int shmem_transport_init(long eager_size)
     if(ret!=0)
         return ret;
 
+#ifdef ENABLE_THREADS
```

Comment:
I don't think ```#ifdef ENABLE_THREADS``` is needed here.  The macros should automatically take care of this.